rest_device.go

```diff
@@ -402,9 +414,15 @@ func createRedundantDeviceRequest(primary Device, secondary Device) api.DeviceRe
 	return req
 }
 
-func (c RestClient) replaceDeviceACLTemplate(uuid string, templateID string) error {
+func (c RestClient) replaceDeviceACLTemplate(uuid string, wanAclTemplateUuid *string, mgmtAclTemplateUuid *string) error {
 	path := "/ne/v1/devices/" + url.PathEscape(uuid) + "/acl"
-	reqBody := api.DeviceACLTemplateRequest{TemplateUUID: &templateID}
+	reqBody := api.DeviceACLTemplateRequest{}
+	if wanAclTemplateUuid != nil {
+		reqBody.TemplateUUID = wanAclTemplateUuid
```

Comment:
The `if` conditions are not needed here. `omitempty` will give the desired behavior.

Reply:
Fixed.